### PR TITLE
Blog post 110 final

### PR DESCRIPTION
**Pull request by matzew**
replacing https://github.com/aerogear/aerogear.org/pull/615

### CODE REVIEW

**Comment by kpiwko** [comment]
@matzew can you update README.md to have correct ruby instructions?

```
rvm install ruby 2.1.7
rvm gemset create aerogear
rvm use ruby-2.1.7@aerogear
bundle install --path vendor
bundle exec jekyll serve --watch
```

I can see that that AEROGEAR issue has been fixed, yay!

However, I had some layout issues in news section - see http://postimg.org/image/65jfe9xkp/777f41f4/

**Comment by matzew** [comment]
k,

let me do that in a different PR - ok ?

On Thu, Nov 5, 2015 at 11:27 AM, Karel Piwko <notifications@github.com>
wrote:

> @matzew <https://github.com/matzew> can you update README.md to have
> correct ruby instructions?
>
> rvm install ruby 2.1.7
> rvm gemset create aerogear
> rvm use ruby-2.1.7@aerogear
> bundle install --path vendor
> bundle exec jekyll serve --watch
>
> I can see that that AEROGEAR issue has been fixed, yay!
>
> However, I had some layout issues in news section - see
> http://postimg.org/image/65jfe9xkp/777f41f4/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/616#issuecomment-154021452>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

File: _posts/2015-11-05-aerogear-unified-push-110-is-out.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+layout: news
+section: news
+title: AeroGear UnifiedPush Server 1.1.0 released!
+author: matzew
+module: push
+releases: 1.1.0.Final
+---
+
+The AeroGear team is happy to announce the version 1.1.0 of the UnifiedPush Server! Over the last year we have developed several new features on our 1.1.0 release series, with a two alpha and four beta releases. The highlight of the release The release focuses on a few new features
```
Comment (sebastienblanc) [inline _posts/2015-11-05-aerogear-unified-push-110-is-out.md]: `The highlight of the release The release focuses on a few new features` There is something wrong here :)

File: _posts/2015-11-05-aerogear-unified-push-110-is-out.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+* Microsoft’s Windows Push Notification service (WNS)
+* Microsoft’s Push Notification service (MPNs)
+* Amazon Device Messaging (ADM) - _experimental_
+* Analytics and Push Metrics
+* an all new Admin UI
```
Comment (sebastienblanc) [inline _posts/2015-11-05-aerogear-unified-push-110-is-out.md]: Let's start with an uppercase

File: _posts/2015-11-05-aerogear-unified-push-110-is-out.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+* Microsoft’s Windows Push Notification service (WNS)
+* Microsoft’s Push Notification service (MPNs)
+* Amazon Device Messaging (ADM) - _experimental_
+* Analytics and Push Metrics
+* an all new Admin UI 
+* JMS based architecutre 
+* Device metadata import
+* Java-8 support
+* new REST API doc (miredot)
```
Comment (sebastienblanc) [inline _posts/2015-11-05-aerogear-unified-push-110-is-out.md]: Let's start with an uppercase

File: _posts/2015-11-05-aerogear-unified-push-110-is-out.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+---
+
+The AeroGear team is happy to announce the version 1.1.0 of the UnifiedPush Server! Over the last year we have developed several new features on our 1.1.0 release series, with a two alpha and four beta releases. The highlight of the release The release focuses on a few new features
+
+* Microsoft’s Windows Push Notification service (WNS)
+* Microsoft’s Push Notification service (MPNs)
+* Amazon Device Messaging (ADM) - _experimental_
+* Analytics and Push Metrics
+* an all new Admin UI 
+* JMS based architecutre
```
Comment (sebastienblanc) [inline _posts/2015-11-05-aerogear-unified-push-110-is-out.md]: `architecture`

**Comment by matzew** [comment]
why not 2.1.x, @kpiwko ?

On Thu, Nov 5, 2015 at 11:27 AM, Karel Piwko <notifications@github.com>
wrote:

> @matzew <https://github.com/matzew> can you update README.md to have
> correct ruby instructions?
>
> rvm install ruby 2.1.7
> rvm gemset create aerogear
> rvm use ruby-2.1.7@aerogear
> bundle install --path vendor
> bundle exec jekyll serve --watch
>
> I can see that that AEROGEAR issue has been fixed, yay!
>
> However, I had some layout issues in news section - see
> http://postimg.org/image/65jfe9xkp/777f41f4/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear.org/pull/616#issuecomment-154021452>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Comment by matzew** [comment]
@sebastienblanc thanks! addressed the comments

**Comment by kpiwko** [comment]
@matzew when I've tried 2.1.x it was not working at my place, rvm not able to download anything

**Comment by kpiwko** [comment]
See here:

```
➜  aerogear.org git:(doc-updates-2) rvm --version
rvm 1.26.11 (latest) by Wayne E. Seguin <wayneeseguin@gmail.com>, Michal Papis <mpapis@gmail.com> [https://rvm.io/]
➜  aerogear.org git:(doc-updates-2) rvm install ruby 2.1.x
Searching for binary rubies, this might take some time.
No binary rubies available for: osx/10.11/x86_64/ruby-2.1.x.
Continuing with compilation. Please read 'rvm help mount' to get more information on binary rubies.
Checking requirements for osx.
Certificates in '/usr/local/etc/openssl/cert.pem' are already up to date.
Requirements installation successful.
Installing Ruby from source to: /Users/kpiwko/.rvm/rubies/ruby-2.1.x, this may take a while depending on your cpu(s)...
ruby-2.1.x - #downloading ruby-2.1.x, this may take a while depending on your connection...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0curl: (22) The requested URL returned error: 404 Not Found
The requested url does not exist(22): 'http://cache.ruby-lang.org/pub/ruby/./ruby-2.1.x.tar.bz2'
Checking fallback: http://ftp.ruby-lang.org/pub/ruby/./ruby-2.1.x.tar.bz2
Checking fallback: http://www.mirrorservice.org/sites/ftp.ruby-lang.org/pub/ruby/./ruby-2.1.x.tar.bz2
No fallback URL could be found, try increasing timeout with:

    echo "export rvm_max_time_flag=20" >> ~/.rvmrc

There has been an error fetching the ruby interpreter. Halting the installation.
```

**Comment by sebastienblanc** [comment]
@kpiwko it should work with `nvm install ruby-2.1.2`

**Comment by kpiwko** [comment]
@sebastienblanc yes, that's why I put 2.1.7 there, as 2.1.x was not working.

**Comment by kpiwko** [comment]
@matzew agreed, let's have another PR for that.

**Pull request merged**